source/includes/steps-install-shell-linux-deb.yaml

```diff
@@ -98,14 +98,14 @@ content: |
 
   .. code-block:: sh
 
-      sudo apt-get install -y mongosh
+      sudo apt-get install -y mongodb-mongosh
   
   Optional. ``apt-get`` will upgrade the packages when a newer version
   becomes available. To prevent unintended upgrades, you can pin the
   package at the currently installed version:
 
   .. code-block:: sh
 
-    echo "mongosh hold" | sudo dpkg --set-selections
+    echo "mongodb-mongosh hold" | sudo dpkg --set-selections
```

Comment:
It's something we mention on the main MongoDB installation docs. Happy to remove 😄

Reply:
@mmarcon Any thoughts? :)

Either way, I think you can go ahead with merging this

Reply:
I'd not include that for `mongosh`. I think we want to users to keep up to date with the new stuff we release rather than encouraging them to pin to the current version.

I can see how that makes sense for the server and for the legacy shell as there is almost a 1:1 relationship between them and new versions are released yearly. However, it's preferable that users keep `mongosh` up to date.